cluster.go

```diff
@@ -406,14 +454,19 @@ func (c *Cluster) doInner(a Action, addr, key string, ask bool, attempts int) er
 	thisA := a
 	if ask {
 		thisA = WithConn(key, func(conn Conn) error {
-			return askConn{conn}.Do(a)
+			err = askConn{conn}.Do(a)
+			c.traceGotResponse(addr, key, attempts, err)
```

Comment:
This will call `traceGotResponse` multiple times won't it? Once here and than below on line 464 or 468. In that case, just remove this.

Reply:
I misunderstood. I should remove this line. Thanks.